connectors/apache-connector/src/main/java/org/glassfish/jersey/apache/connector/ApacheClientProperties.java

```diff
@@ -156,6 +156,19 @@
      */
     public static final String KEEPALIVE_STRATEGY = "jersey.config.apache.client.keepAliveStrategy";
 
+    /**
+     * RedirectStrategy which can customize how redirects are handled.
+     * <p/>
+     * The value MUST be a subclass of {@link RedirectStrategy}.
+     */
+    public static final String REDIRECT_STRATEGY = "jersey.config.apache.client.redirectStrategy";
+
+    /**
+     * DefaultAuthSchemeRegistry which can customize the auth scheme used.
+     * <p/>
+     * The value MUST be a subclass of {@link org.apache.http.config.Registry}.
+     */
+    public static final String DEFAULT_AUTH_SCHEME_REGISTRY = "jersey.config.apache.client.defaultAuthSchemeRegistry";
```

Comment:
I would advice to rename the ``DEFAULT_AUTH_SCHEME_REGISTRY`` to ``AUTH_SCHEME_REGISTRY`` which later becomes default for Apache connector (by set default..() method in builder). However for us default means that already there is some predefined registry which can be set automatically if there is nothing else. However in this case there is no such a Registry predefined.